front/public/locales/fr/errors.json

```
@@ -233,7 +233,15 @@
       "InfraNotFound": "Infrastructure '{{infra_id}}' non trouvée",
       "InfraNotLoaded": "L'infrastructure '{{infra_id}}' n'est pas chargée",
       "NotFound": "Grille horaire '{{timetable_id}}' non trouvée",
-      "ParseError": "Impossible de parser train_id : '{{train_id}}'"
+      "ParseError": "Impossible de parser train_id : '{{train_id}}'",
+      "similar_trains": {
+        "Database": "Erreur interne (base de données)",
+        "InfraNotFound": "Infrastructure '{{infra_id}}' non trouvée",
+        "InvalidPath": "Chemin invalide dans la circulation",
+        "RollingStockNotFound": "Matériel roulant '{{rolling_stock_name}}' non trouvé",
+        "SpeedLimitNotFound": "Tag de limite de vitesse '{{speed_limit_tag}}' non trouvé",
```

We usually translate speed limit tag by this
```suggestion
        "SpeedLimitNotFound": "Code de composition '{{speed_limit_tag}}' non trouvé",
```

---

Of course 🤦 https://github.com/OpenRailAssociation/osrd/compare/c74be5df9e30e94a51324c6939f899882a9bbc8d..50fef39e5dc4b7c3d4d5c6bcf07faa8986acf4ac